### PR TITLE
Add cluster upgrade freeze window model

### PR DESCRIPTION
**Pull request by tzvatot**
@igoihman @zgalor PTAL

### CODE REVIEW

File: model/clusters_mgmt/v1/freeze_window_type.model [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of a cluster upgrade freeze window that can be set globally for cluster upgrades.
+class ClusterUpgradeFreezeWindow {
+	// Start is the start time of the upgrade freeze window
```
Comment (zgalor) [inline model/clusters_mgmt/v1/freeze_window_type.model:19]: Comment should end with `.`

File: model/clusters_mgmt/v1/freeze_window_type.model [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	Start Date
+
+	// End is the end time of the upgrade freeze window
```
Comment (zgalor) [inline model/clusters_mgmt/v1/freeze_window_type.model:22]: comment should end with `.`

**Comment by zgalor** [comment]
Where is this new type used?

**Comment by tzvatot** [comment]
Pushed back, no freeze window at this time.

**Pull request closed without merging**